ui/app/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
Might be easier to publish both back-end and front-end under the same license? What do you think @brancz ?